osu.Framework/Graphics/Containers/RearrangeableListContainer.cs

```diff
@@ -110,7 +111,11 @@ private void removeItems(IList items)
                 if (currentlyDraggedItem != null && EqualityComparer<TModel>.Default.Equals(currentlyDraggedItem.Model, item))
                     currentlyDraggedItem = null;
 
-                ListContainer.Remove(itemMap[item]);
+                var drawableItem = itemMap[item];
+
+                ListContainer.Remove(drawableItem);
+                AsyncDisposalQueue.Enqueue(drawableItem);
```

Comment:
Out of curiosity I've looked at who else calls this to check if there are any other existing scenarios in framework, and stumbled upon `CompositeDrawable.DisposeChildAsync()` which also enqueues for async disposal, but also calls `UnbindAllBindablesSubTree()` on the disposed drawable. Therefore I think that would be an even better thing to call here?

```suggestion
                DisposeChildAsync(drawableItem);
```